test/cluster/test_raft_recovery_user_data.py

```diff
@@ -74,7 +74,9 @@ async def test_raft_recovery_user_data(manager: ManagerClient, remove_dead_nodes
 
     rf: int = 3
     ks_name = unique_name()
-    finish_writes = await start_writes(cql, rf, ConsistencyLevel.LOCAL_QUORUM, concurrency=5,
+    # FIXME: Consider increasing concurrency (here and in write workloads below) once
+    # https://github.com/scylladb/scylladb/issues/23565 is fixed.
+    finish_writes = await start_writes(cql, rf, ConsistencyLevel.LOCAL_QUORUM, concurrency=1,
                                        ks_name=ks_name, node_shutdowns=True)
```

Comment:
If the goal is to deflake the test, did you consider disabling hinted handoff instead? This is controlled by the `hinted_handoff_enabled` configuration option.